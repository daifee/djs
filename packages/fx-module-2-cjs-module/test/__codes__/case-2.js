/**
 * 合唱
 */
fxDefine('chorus', () => {


  __inline('/static/room/modules/chorus/js/block.es6');

  const blockUtil = fxRequire('chorus/block');


  __inline('/static/room/modules/chorus/js/handler.es6');
  __inline('/static/room/modules/chorus/sort/pk.es6');
  const chorusPk = fxRequire('chorus/pk');

  __inline('/static/room/modules/chorus/sort/song.es6');
  const chorusSong = fxRequire('chorus/song');

  const {
    getChorusConfig
  } = fxRequire('chorus/service');
  const { showToast } = fxRequire('chorus/toast');

  let songName = null;

  const {
    EVENT_ID,
  } = fxRequire('chorus/const');

  const DEFAULT_CFG = {
    confirmTimeout: 20,
    endDelayTime: 20
  }

  const Api = {

  }
  return Api
})
