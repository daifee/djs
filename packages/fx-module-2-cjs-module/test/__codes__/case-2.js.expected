"use strict";

/**
 * 合唱
 */

__inline('/static/room/modules/chorus/js/block.es6');
const blockUtil = require('chorus/block');
__inline('/static/room/modules/chorus/js/handler.es6');
__inline('/static/room/modules/chorus/sort/pk.es6');
const chorusPk = require('chorus/pk');
__inline('/static/room/modules/chorus/sort/song.es6');
const chorusSong = require('chorus/song');
const {
  getChorusConfig
} = require('chorus/service');
const {
  showToast
} = require('chorus/toast');
let songName = null;
const {
  EVENT_ID
} = require('chorus/const');
const DEFAULT_CFG = {
  confirmTimeout: 20,
  endDelayTime: 20
};
const Api = {};
module.exports = Api